cms/static/js/views/modals/course_outline_modals.js

```diff
@@ -389,6 +389,58 @@ define(['jquery', 'backbone', 'underscore', 'gettext', 'js/views/baseview',
         }
     });
 
+    SelfPacedDueDateEditor = AbstractEditor.extend({
+        fieldName: 'due_num_weeks',
+        templateName: 'self-paced-due-date-editor',
+        className: 'modal-section-content has-actions due-date-input grading-due-date',
+
+        events: {
+            'click .clear-date': 'clearValue',
+            'keyup #due_in': 'validateDueIn',
+            'blur #due_in': 'validateDueIn',
+        },
+
+        getValue: function() {
+            return parseInt(this.$('#due_in').val());
+        },
+
+        validateDueIn: function() {
+            if (this.getValue() > 18){
+                this.$('#warning').show();
```

Comment:
If we are going to do it this way, I'd say to rename this to `warning-max`. Also not sure if there are other thoughts here or if this is the desired way, but I thought we typically wanted to avoid using ids. Also also, just now realizing that we should probably be a bit more specific with our naming here. `warning` could be common in the whole document. Maybe update to something like `due-num-weeks-warning-max`/`due-num-weeks-warning-min`